app/serializers/show_book_serializer.rb

```diff
@@ -0,0 +1,8 @@
+class ShowBookSerializer < BookBaseSerializerSerializer
+  attributes :image, :actual_rent
+  def actual_rent
+    rents = object.rents.to_a
```

Comment:
Like I told you yesterday, this should not be called with to_a, this will load everything to memory.
I recommend to do something like: 
```
last = object.rents.last
last if Time.zone.now < rent.to
```

We then to use Time.zone.now instead of Date.current